lightning/src/ln/channelmanager.rs

```diff
@@ -10901,9 +10900,8 @@ macro_rules! create_offer_builder { ($self: ident, $builder: ty) => {
 	///
 	/// # Privacy
 	///
-	/// Uses [`MessageRouter`] to construct a [`BlindedMessagePath`] for the offer based on the given
-	/// `absolute_expiry` according to [`MAX_SHORT_LIVED_RELATIVE_EXPIRY`]. See those docs for
-	/// privacy implications.
+	/// Uses the [`ChannelManager`]'s [`MessageRouter`] to construct a [`BlindedMessagePath`] for
```

Comment:
Won't this use `OffersMessageFlow`'s message router? Seems like `ChannelManager` no longer has one as it is being passed through to `OffersMessageFlow`. Since we are only creating the flow internally now, perhaps we should word this as:

```rust
/// Uses the [`MessageRouter`] passed to [`ChannelManager`]'s constructor to ...
```